common/kernel/property.h

```diff
@@ -42,7 +42,7 @@ struct Property
     };
 
     Property();
-    Property(int64_t intval, int width = 32);
+    Property(int64_t intval, int width = 64);
```

Comment:
I'm concerned this might end up having unwanted consequences elsewhere - perhaps you could just instantiate it 64 bits wide in the one place where seed is being saved?